docs/user_authentication/third_party_token_management.md

```diff
@@ -1,42 +1,58 @@
 
 ```sequence {theme: 'simple'}
-title Third Party IDP Token Management
+title Third party token management
```

Comment:
For titles, I'm a fan of capitalization; also, I think "third-party" should be hyphenated:
```
title Third-party Token Management
```